release.yaml

```diff
@@ -476,19 +476,19 @@ managers:
 
   - artifact: dev.galasa.windows.manager
     version: 0.21.0
```

Comment:
Not sure what the windows manager does ? You are pulling it out of everywhere intentionally ?

Reply:
Thank you for noticing I had in my notes in the README.md changes I made earlier this year that the windows manager looked minimally implemented so thought it best to remove but I've just found when looking again that its a dependency of another two managers, so I've added it back into the OBR, BOM and Isolated as those two managers are in isolated and all dependencies need to be.